pkg/capabilities/mercury/mercury.go

```diff
@@ -24,6 +30,11 @@ func (id FeedID) String() string {
 	return string(id)
 }
 
+func (id FeedID) Bytes() [32]byte {
+	b, _ := hex.DecodeString(string(id)[2:])
+	return [32]byte(b)
```

Comment:
This will panic it b is shorter than 32 bytes - is that ok or should we pad with zeros (only to prevent a panic)?